src/BlockEntities/FurnaceEntity.cpp

```diff
@@ -189,6 +189,7 @@ void cFurnaceEntity::BurnNewFuel(void)
 	// Is the input and output ready for cooking?
 	if (!CanCookInputToOutput())
```

Comment:
Maybe merge this condition with the one above, so that both BurnTimes and IsCooking are set?